ledger/db/src/lib.rs

```diff
@@ -2528,7 +2528,7 @@ mod ledger_db_test {
             ..Default::default()
         };
         let block = Block::new_with_parent(
-            BlockVersion::ONE,
+            BlockVersion::ZERO,
```

Comment:
How about we use the `BLOCK_VERSION` const throughout this file?

Reply:
Will fix